starlingcaptureapi/exif.py

```diff
@@ -56,6 +56,46 @@ def convert_timestamp(self, value_str):
         """
         return parser.parse(value_str).strftime("%Y:%m:%d %H:%M:%S %z")
 
+    def extract_from_exif_data(self, exif_data):
```

Comment:
Since this returns a lat/lon, you could call this `lat_lon_from_exif` or something like that.